src/circuit/dialer.js

```diff
@@ -0,0 +1,186 @@
+'use strict'
+
+require('safe-buffer')
```

Comment:
both set `immediate` and `safe-buffer` should autopolyfill depending on environment and available features, any reason why we wouldn't want to rely on that?